rocky/rocky/views/upload_csv.py

```diff
@@ -95,6 +99,8 @@ def get_or_create_reference(self, ooi_type_name: str, value: str):
         return ooi
 
     def get_ooi_from_csv(self, ooi_type_name: str, values: Dict[str, str]):
+        ckey = "clearance"
+        level = int(values[ckey]) if ckey in values and values[ckey] in list("01234") else None
```

Comment:
Two little things:
- `key` would suffice as a name; and if you really must be that specific then `clearance_key` would be appropriate as well
- You might want to extract this literal to a constant outside of the function. E.g. as `CLEARANCE_VALUES = list("01234")`. This would make it a bit more clear and maintainable. Also for performance reasons you'd want to use literals, which `"01234"` is, but the extra `list` call is not.